src/pyscipopt/scip.pxi

```diff
@@ -5031,4 +5081,4 @@ def is_memory_freed():
     return BMSgetMemoryUsed() == 0
 
 def print_memory_in_use():
-    BMScheckEmptyMemory()
+    BMScheckEmptyMemory()
```

Comment:
I don't get what happened here, how was this line deleted before?

Reply:
shall we add a test for this too? just something that calls this method should be enough.

Reply:
Yeah, I have no idea, Mo, I don't think I messed with that. But I'm all for adding more tests :)

Reply:
But it is a bit strange, it's not a method of any class, it might have been for debugging purposes only. Added the test either way.